pipelinesScripts/validator/validator.go

```diff
@@ -4,21 +4,27 @@ import (
 	"errors"
 	"fmt"
 	"io/ioutil"
-	"log"
 	"os"
-	"os/exec"
+	"path/filepath"
 	"strings"
 
+	"github.com/jfrog/jfrog-cli-plugins-reg/dependency"
+	"github.com/jfrog/jfrog-cli-plugins-reg/git"
+	"github.com/jfrog/jfrog-cli-plugins-reg/github"
 	"github.com/jfrog/jfrog-cli-plugins-reg/utils"
 )
 
+const (
+	GitHubIssueTitle = "Failed upgrading dependencies"
+	GitHubIssueBody  = "This issue opened by the JFrog CLI plugins bot. I attended to upgrade the following plugin(s) to\n%s:%s\nThe following commands failed after upgrading:\n go ver ./...\ngo test -v ./...\nThe upgrade was therefore aborted. Please fix and upgrade manually."
```

Comment:
This issue opened by the JFrog CLI plugins bot. I attended to upgrade the following plugin(s) to\n%s:%s\nThe following commands failed after upgrading:\n go ver ./...\ngo test -v ./...\nThe upgrade was therefore aborted. Please fix and upgrade manually."
-->
This issue was opened by the JFrog CLI Plugins Registry bot. We attended to upgrade the following plugin(s) to\n%s:%s\nThe following commands failed after upgrading:\n go vet -v ./...\ngo test -v ./...\nThe upgrade commit and push were therefore aborted. Please fix the issue and upgrade manually."